### PR TITLE
feat: Add custom theme file

### PR DESCRIPTION
**Pull request by michaelbukachi**
## What is the Purpose?
Add custom theme file

## What was the approach?
Add new .scss file filled with a couple of variables
Corrected font issue

## Are there any concerns to addressed further before or after merging this PR?
State some additional info if any. For instance running `install` or setting some environment variable(s)

## Mentions?
@andrewtpham @ninetteadhikari 

## Issue(s) affected?
#74

### CODE REVIEW

File: src/styles/theme.scss [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+$danger-color: $red ;
+$main-font-family: 'Lato', sans-serif;
+$primary-background-color: #F5F8FA;
+$light-color: $gray-very-light;
+$dark-color: $gray;
+
+
+/* Title - Lato B48 */
+
+.title {
```
Comment (andrewtpham) [inline src/styles/theme.scss:40]: @michaelbukachi With how you have set this up does this mean we can call `.title` in our <style> section of our components to use the features under style? If not, is there a way to be able to define something like `main-font-family-title` as a variable and call it across the app?

File: src/styles/theme.scss [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  font-style: normal;
+  font-weight: bold;
+  font-size: 35px;
+  line-height: 42px;
+  color: #C4C4C4;
+}
+
+/* Body 2 Bold - Lato B22 */
+
+.heading {
```
Comment (andrewtpham) [inline src/styles/theme.scss:62]: Should this be something like `.body-bold`?

File: src/styles/theme.scss (hunk 1 above)
Comment (michaelbukachi) [reply src/styles/theme.scss:40]: To apply styling for the `title`, setting the class of a tag to `title` should do it. e.g `<span class="title">Some tile</span>`

File: src/styles/theme.scss (hunk 2 above)
Comment (michaelbukachi) [reply src/styles/theme.scss:62]: Oops. Yes, I just copied the css in the issue so I didn't take not of the comments.

Comment (andrewtpham) [reply src/styles/theme.scss:62]: ok no worries, I'll update it when I play around with colors :)

**Pull request merged**